vlog4j-core/src/test/java/org/semanticweb/vlog4j/core/model/TermImplTest.java

```diff
@@ -132,6 +132,42 @@ public void namedNullGetterTest() {
 		assertEquals(TermType.NAMED_NULL, n.getType());
 	}
 
+	@Test
```

Comment:
careful with method names.

Reply:
This is not my test, I did not implement this.